amico/core.py

```diff
@@ -290,10 +316,6 @@ def fit( self ) :
                     if self.scheme.b0_count > 0 :
```

Comment:
Do we need to compute the mean b0 here? As you removed the following lines (293-296) because indeed the normalization is done before once and for all, perhaps we may compute the b0 only later on at line 241 if needed, no?

Also, if the data is normalized, the mean is 1, no? So, in order not to break the following code starting at line 334 (`if self.get_config('doSaveCorrectedDWI') :`), perhaps we have to calculate and store in a variable the values of the b0s for later usage. What do you think